README.md

```diff
@@ -592,8 +592,59 @@ drep_xsk1vpdsm49smzmdwhd4kjmm2mdyljjysm746rafjr7r8kgfanj849psw8pfm305g59wng0akw3
 $ cardano-address key public --with-chain-code < drep.xsk > drep.xvk
 drep_xvk1mg7xae48d7z4nntd35tey0jmclxaavwmk3kw2lkkt07p3s3x3yy45805manx2kj2neg40kfpy9em36vnkjfm4fw09k66837unrvd70qq8ewzf
 
-$ cardano-address key hash < drep.xvk > drep
+$ cardano-address key public --without-chain-code < drep.xsk > drep.vk
+drep_vk1mg7xae48d7z4nntd35tey0jmclxaavwmk3kw2lkkt07p3s3x3yysra6588
+
+$ cardano-address key hash < drep.xvk > drep.vkh
+drep_vkh1sp5xhvmj0asztqfsjyta3cwvq7jppc2rwmfcsggp62va5hqgpg4
+
+$ cat drep.vkh | bech32
+80686bb3727f602581309117d8e1cc07a410e14376d3882101d299da
+
+$ cat drep.vkh | bech32 drep > drep.deprecated
+drep1sp5xhvmj0asztqfsjyta3cwvq7jppc2rwmfcsggp62va538nup0
+
+$ cat drep.deprecated | bech32
+80686bb3727f602581309117d8e1cc07a410e14376d3882101d299da
+
+$ cardano-address key hash --with-byte < drep.xvk > drep.credential
+drep1y2qxs6anwflkqfvpxzg30k8pesr6gy8pgdmd8zppq8ffnksapjznm
+
+$ cat drep.credential | bech32
+2280686bb3727f602581309117d8e1cc07a410e14376d3882101d299da
```

Comment:
> $ cardano-address key hash --with-byte < drep.xvk > drep.credential
drep1y2qxs6anwflkqfvpxzg30k8pesr6gy8pgdmd8zppq8ffnksapjznm

Apologies for being a bit late to the game here, but the `CIP-105` version is deprecated and the `CIP-129` version should be considered default. Does it make more sense to move the optional function modifier flag (in this case `--with-byte`) to instead be present to trigger the "Legacy" or "Deprecated" mode and return the `CIP-105` version?

Reply:
Example, instead of requiring the `--with-byte` flag to generate the most-current version of the expected output, we could instead add a `--cip-105` or `--legacy` flag to generate the deprecated model?

Reply:
good suggestion. As mentioned above we have `--cip-0105` flag now on demand